mintlify/snippets/kyc-onboarding.mdx

```diff
@@ -0,0 +1,632 @@
+
+{
+  "umaAddress": "$john.doe@mycompany.com",
+  "platformCustomerId": "customer_12345",
+  "customerType": "INDIVIDUAL",
+  "fullName": "John Michael Doe",
+  "birthDate": "1990-01-15",
+  "nationality": "US",
+  "address": {
+    "line1": "123 Main Street",
+    "line2": "Apt 4B",
+    "city": "San Francisco",
+    "state": "CA",
+    "postalCode": "94105",
+    "country": "US"
+  },
+  "bankAccountInfo": {
+    "accountType": "US_ACCOUNT",
+    "accountNumber": "123456789",
+    "routingNumber": "987654321",
+    "accountCategory": "CHECKING",
+    "bankName": "Chase Bank"
+  }
+}
+```
+</RequestExample>
+
+<ResponseExample>
+```json Success
+{
+  "id": "Customer:019542f5-b3e7-1d02-0000-000000000001",
+  "umaAddress": "$john.doe@mycompany.com",
+  "platformCustomerId": "customer_12345",
+  "customerType": "INDIVIDUAL",
+  "fullName": "John Michael Doe",
+  "birthDate": "1990-01-15",
+  "nationality": "US",
+  "address": {
+    "line1": "123 Main Street",
+    "line2": "Apt 4B",
+    "city": "San Francisco",
+    "state": "CA",
+    "postalCode": "94105",
+    "country": "US"
+  },
+  "kycStatus": "APPROVED",
+  "createdAt": "2023-07-21T17:32:28Z",
+  "updatedAt": "2023-07-21T17:32:28Z",
+  "isDeleted": false
+}
+```
+</ResponseExample>
+
+### Create Business Customer
+
+<RequestExample>
+```http POST /customers
+POST /customers
+Authorization: Basic <base64-encoded-credentials>
+Content-Type: application/json
+
+{
+  "umaAddress": "$acme.corp@mycompany.com",
+  "platformCustomerId": "business_67890",
+  "customerType": "BUSINESS",
+  "address": {
+    "line1": "456 Oak Avenue",
+    "line2": "Floor 12",
+    "city": "New York",
+    "state": "NY",
+    "postalCode": "10001",
+    "country": "US"
+  },
+  "businessInfo": {
+    "legalName": "Acme Corporation, Inc.",
+    "registrationNumber": "BRN-123456789",
+    "taxId": "EIN-987654321"
+  },
+  "bankAccountInfo": {
+    "accountType": "US_ACCOUNT",
+    "accountNumber": "987654321",
+    "routingNumber": "123456789",
+    "accountCategory": "CHECKING",
+    "bankName": "Bank of America"
+  }
+}
+```
+</RequestExample>
+
+<ResponseExample>
+```json Success
+{
+  "id": "Customer:019542f5-b3e7-1d02-0000-000000000002",
+  "umaAddress": "$acme.corp@mycompany.com",
+  "platformCustomerId": "business_67890",
+  "customerType": "BUSINESS",
+  "address": {
+    "line1": "456 Oak Avenue",
+    "line2": "Floor 12",
+    "city": "New York",
+    "state": "NY",
+    "postalCode": "10001",
+    "country": "US"
+  },
+  "businessInfo": {
+    "legalName": "Acme Corporation, Inc.",
+    "registrationNumber": "BRN-123456789",
+    "taxId": "EIN-987654321"
+  },
+  "kycStatus": "APPROVED",
+  "createdAt": "2023-07-21T17:32:28Z",
+  "updatedAt": "2023-07-21T17:32:28Z",
+  "isDeleted": false
+}
+```
+</ResponseExample>
+
+## Schema References
+
+The customer creation endpoints reference the following OpenAPI schema components:
+
+<AccordionGroup>
+<Accordion title="Customer Schema">
```

Comment:
Is there really much value in this section? It kind of feels like a bit of a duplicate of the API reference for these objects. Maybe it's helpful though...